openapi.yaml

```diff
@@ -0,0 +1,404 @@
+swagger: "2.0"
+info:
+  description: "reader api service"
+  version: "0.0.1"
+  title: "Reader API"
+  contact:
+    email: "hello@gorillab.co"
+host: "glab-reader-api.herokuapp.com"
+basePath: "/api"
+tags:
+- name: "auth"
+  description: "Auth module"
+- name: "user"
+  description: "User module"
+- name: "source"
+  description: "Source module"
+- name: "post"
+  description: "Post module"
+schemes:
+- "http"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /auth/facebook:
+    get:
+      tags:
+      - "auth"
+      summary: "Logs user into the system by facebook"
+      description: ""
+      operationId: ""
+      responses:
+        200:
+          description: "successful operation, send a facebook login form"
```

Comment:
this endpoint does not response 200. it will redirect user to facebook login page so it should be `301`. check this for more info https://en.wikipedia.org/wiki/HTTP_301